app/reducers/snackbar.js

```diff
@@ -214,6 +221,10 @@ const messages = defineMessages({
   STOPTICKETBUYERV2_SUCCESS: {
     id: "stopTicketBuyer.Success",
     defaultMessage: "Ticket Buyer successfully stopped."
+  },
+  WRONG_PASSPHRASE_MSG: {
+    id: "errors.wrongPassphrase",
+    defaultMessage: "Wrong private passphrase entered. Please verify you have typed the correct private passphrase for the wallet."
```

Comment:
Everything looks good, but I'm seeing this message get cut in the middle of a "the".  So it show t on the top row and he on the bottom.  Maybe needs to have a different word wrap on the snackbar div?

Reply:
Yeah, that's one of the things I'm planning to address on a separate snackbar PR (fix the word wrap).

This is likely to need changes in the individual message components of the snackbar, so I wanted to leave it for another PR, but if you don't have objections I can add another commit here for it.

Reply:
Ah gotcha. Cool